mmf/utils/logger.py

```diff
@@ -98,10 +98,11 @@ def setup_logger(
     distributed_rank = get_rank()
     handlers = []
 
+    logging_level = registry.get("config").training.logger_level.upper()
     if distributed_rank == 0:
-        logger.setLevel(logging.INFO)
+        logger.setLevel(logging_level)
```

Comment:
Does it work directly, don't you have to do `getattr(logging, logging_level)`?

Reply:
I tested and verified that it works. In https://docs.python.org/3/library/logging.html#logging.Logger.setLevel, the input can be a str:
> The level parameter now accepts a string representation of the level such as ‘INFO’ as an alternative to the integer constants such as "INFO".